src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Public.Data/ReleasesController.cs

```diff
@@ -0,0 +1,29 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Public.Data;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Public.Data;
+
+[Authorize]
+[ApiController]
+[Route("api/public-data/releases")]
```

Comment:
I wonder if the `DataSetsController` is a better place for this to live rather than `ReleasesController`.

The endpoint could be `api/public-data/data-set-candidates?releaseId={releaseId}`

One to chat through with the team I think.

Reply:
moved to `DataSetCandidatesController` with the suggested route